src/lib/components/consignments/setup/info.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import Button from "../../buttons/inverted_button"
+import Welcome from "./welcome"
+
+import {
```

Comment:
TSLint should be catching this - will look to see what's up with this, I don't wanna be remembering this kind of thing 🍡